firestore-next/test.firestore.js

```diff
@@ -1090,6 +1090,77 @@ describe("firestore", () => {
                   limit(25));
               // [END paginate]
             });
+
+            it("should handle OR queries", async () => {
+              const { collection, query, where, and } = require("firebase/firestore");
+              // [START or_query]
+              const query = query(collection(db, "cities"), and(
```

Comment:
Nit: I got some feedback that using `const query = query(...` confuses readers. Here and in the other web snippets, can we change to `const q = query(..`

Reply:
Done.